lib/bandit/pipeline.ex

```diff
@@ -45,13 +45,14 @@ defmodule Bandit.Pipeline do
             Bandit.Telemetry.stop_span(span, adapter.metrics, %{conn: conn})
             {:upgrade, adapter.transport, protocol, opts}
         end
-      rescue
-        error -> handle_error(error, __STACKTRACE__, transport, span, opts)
```

Comment:
I know it's redundant with the catch block below (`raise` being realized by try/catch internally), but I don't *think* it's part of official spec; can we leave this clause here to make it explicit? Alternately, if the language *is* authoritative on this, that's fine too.

Reply:
Right.

Despite working like that doesn't seem to be actually documented in https://hexdocs.pm/elixir/try-catch-and-rescue.html that you can catch all (including `:error`) with just `catch`.

Re-instating the `rescue` clause just in case.

Reply:
Good call. It sounds like it's an implementation detail then, so we're safer to be explicit in case they ever change it in the language.